### ISSUE TITLE
Latest Windows binary release segmentation fault

### ISSUE THREAD

**Issue by sfhacker** [clibs/clib#196]
Ciao.

Just downloaded the latest Windows binary release from here: https://github.com/clibs/clib/releases.
After unzipping the downloaded file, I type: clib-init.exe and I get this exception:

![image](https://user-images.githubusercontent.com/25817529/89589916-15bcc280-d88a-11ea-8d51-fa3b1aba153b.png)

Instead of releasing new versions we should focus of fixing the open issues!
Before releasing a new version, we should run as many tests as possible. (Best practices!)
I know nothing about the code base. However I can contribute with testing, documentation and so on.

Environment: Windows 10 x64

**Comment by stephenmathieson** [comment]
I agree that we should be testing the binaries before releasing. Any and all help would be appreciated!

**Comment by sfhacker** [comment]
@stephenmathieson Releasing segmentation faults is indeed embarrassing and **puts people off**!
What is more, it makes this project **unreliable** and **unpopular**.
I repeat myself: I know nothing about the codebase as to fix runtime exceptions! However, I am more than happy to test whatever fix you apply!
Can you find out why this segmentation fault occurs and let me know so I can test it?

**Comment by stephenmathieson** [comment]
I agree. Broken releases are not going to convince new users to stay, or old users to continue using our software.

Unfortunately I don't have much time to dedicate to this project these days, as I'm no longer using C at my 9-5. I may be able to take a look this weekend, but no promises.

**Comment by Isty001** [comment]
It's a wild guess, but isn't this issue related to #116 ? It looks like you are not invoking clib from your PATH. Could you, (or someone else) check out if clib works correctly on Windows if all the executables are in the PATH?

**Comment by jwerle** [comment]
I'll test **again** on my Windows machine some time this week when I can

**Comment by nouwaarom** [comment]
I believe that #269 fixed this problem.

**Comment by jwerle** [comment]
> I believe that #269 fixed this problem.

closing for now
